### PR TITLE
Add attend and unattend command

### PR DESCRIPTION
**Pull request by Leumu**
- Add attend command to mark the attendance of the student
- Add unattend command to unmark the attendance of the student
- Add tutorial classes from tutorial 1 to tutorial 11
- Update parser to handle edge cases and invalid input gracefully
- Add relevant test cases for both attend and unattend commands

### CODE REVIEW

File: src/main/java/seedu/address/logic/parser/AddressBookParser.java
Comment (salmonkarp) [inline src/main/java/seedu/address/logic/parser/AddressBookParser.java]: Need to add this `AttendCommand.COMMAND_WORD, UnattendCommand.COMMAND_WORD` to allow fuzzyMatch() to succeed when parsing the command. Currently it will always fail (doesn't recognise the command)

File: src/test/java/seedu/address/model/person/TutorialClassTest.java [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+    @Test
+    public void isValidTutorialClass_invalidTutorialClasses_returnFalse() {
+        assertFalse(TutorialClass.isValidTutorialClass("t0"));
+        assertFalse(TutorialClass.isValidTutorialClass("t12"));
+        assertFalse(TutorialClass.isValidTutorialClass("T1")); // case-sensitive
+        assertFalse(TutorialClass.isValidTutorialClass("tutorial1"));
+        assertFalse(TutorialClass.isValidTutorialClass(""));
+        assertFalse(TutorialClass.isValidTutorialClass(" "));
+        assertFalse(TutorialClass.isValidTutorialClass(null));
+    }
```
Comment (BarneyLaw) [inline src/test/java/seedu/address/model/person/TutorialClassTest.java]: in the future we could perhaps have a less restrictive way of allowing how user can define a tutorial. e.g. T1 is still t1, Tutorial 1 will still refer to t1
